src/main/java/org/terasology/climateConditions/visualization/ShowMapCommand.java

```diff
@@ -0,0 +1,121 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.climateConditions.visualization;
+
+import com.google.common.base.Function;
+import org.terasology.climateConditions.ClimateConditionsSystem;
+import org.terasology.climateConditions.ConditionsBaseField;
+import org.terasology.context.Context;
+import org.terasology.entitySystem.systems.BaseComponentSystem;
+import org.terasology.entitySystem.systems.RegisterSystem;
+import org.terasology.logic.console.commandSystem.annotations.Command;
+import org.terasology.logic.console.commandSystem.annotations.CommandParam;
+import org.terasology.logic.players.LocalPlayer;
+import org.terasology.registry.In;
+import org.terasology.rendering.nui.NUIManager;
+import org.terasology.world.WorldProvider;
+
+@RegisterSystem
+public class ShowMapCommand extends BaseComponentSystem {
+    public static final int SIZE_OF_IMAGE = 300;
+
+    @In
+    private ClimateConditionsSystem climateConditions;
+    @In
+    private LocalPlayer localPlayer;
+    @In
+    private NUIManager nuiManager;
+    @In
+    private Context context;
+    @In
+    private WorldProvider worldProvider;
+
+    private ConditionsBaseField base;
+    private int mapHeight;
+
+    @Override
+    public void postBegin() {
+        context.put(ShowMapCommand.class, this);
+    }
+
+    /**
+     * Displays the selected map at the selected height level.
+     *
+     * The lighter a pixel on the map, the higher temperature/humidity/whatever else.
+     * @param mapType
+     */
+    @Command(shortDescription = "Display condition map", helpText = "Show a given map (humidity, temperature) at a given height level.")
+    public void showMap(@CommandParam("map type") String mapType, @CommandParam("height to look at") int height) {
+        mapHeight = height;
+
+        if (climateConditions.getWorldSeed() == null) {
+            setClimateSeed();
+        }
+        if (climateConditions.getTemperatureBaseField() == null) {
+            initializeClimateTemperature();
+        }
+        if (climateConditions.getHumidityBaseField() == null) {
+            initializeClimateHumidity();
+        }
+
+        if (mapType.equals("temperature")) {
+            base = climateConditions.getTemperatureBaseField();
+        } else {
+            base = climateConditions.getHumidityBaseField();
+        }
+
+        nuiManager.pushScreen("ClimateConditions:displayConditionScreen");
+    }
+
+    /**
+     * Configures humidity.
+     */
+    // TODO: change the initialization so that maps reflect the corresponding world location
+    private void initializeClimateHumidity() {
+        climateConditions.configureHumidity(0, 200, 10, new Function<Float, Float>() {
+            @Override
+            public Float apply(Float input) {
+                return input;
+            }
+        }, 0, 1);
```

Comment:
Can't we use lambda expressions here?

```suggestion
        climateConditions.configureHumidity(0, 200, 10, input -> input, 0, 1);
```